### PR TITLE
Fix Telemeter integration-v2 tests

### PR DESCRIPTION
**Pull request by metalmatze**
Something is broken and I'll try to fix it in this PR. I'll get messy, but I'll try to squash it before merging. :upside_down_face:

### CODE REVIEW

**Comment by s-urbaniak** [comment]
thank you! :tada:

File: test/integration-v2.sh [hunk 1]
```diff
@@ -48,6 +48,7 @@ done
     --listen localhost:9103 \
     --listen-internal localhost:9104 \
     --forward-url=http://localhost:9105/api/v1/receive \
+    --memcached=localhost:11211 \
```
Comment (squat) [inline test/integration-v2.sh]: how did you add this line? this is already in the file: https://github.com/openshift/telemeter/blob/master/test/integration-v2.sh#L51

Are you working on the latest of the master branch?

Comment (squat) [reply test/integration-v2.sh]: ah nvm, you removed it and added it back

**Comment by metalmatze** [comment]
The actual fix is in #353.

**Pull request closed without merging**